### ISSUE TITLE
[Quirk] Rounded edges on Xfce4 panels

### ISSUE THREAD

**Issue by kutulhu** [dracula/gtk#270]
Hello, I'd like to know if there is a way to make the panels square so they don't leave blank spaces, attached pic, see panel edges and networking icon for reference.
![Screenshot 2024-04-08 173341](https://github.com/dracula/gtk/assets/16739139/9d62a5f2-9300-48bc-b432-252ec7aae710)

**Comment by kutulhu** [comment]
Nevermind (kind of) managed to "fix it" by editing xfce4's gtk.css, I'm kinda new with linux so I don't know if this is the correct way to do it, but I'll leave the "fix" and close this if this is fine as solution.

edit ~/.config/gtk-3.0/gtk.css
add the following

```
.xfce4-panel {
   border-bottom-left-radius: 10px;
   border-bottom-right-radius: 10px;
   border-top-left-radius: 10px;
   border-top-right-radius: 10px;
}
```

restart the panel with 

`xfce4-panel -r`

hope this helps for anyone running into the same thing.

**Comment by EliverLara** [comment]
Thanks for reporting! it was fixed with 6ed4a6dfe6579a409dafbfe48a5e7473eab2a4bc
